build.gradle.kts

```diff
@@ -12,6 +12,9 @@ tasks.register("clean", Delete::class) {
 }
 
 buildscript {
+    dependencies {
+        classpath(libs.ossLicensesPlugin)
+    }
```

Comment:
Please refer here to add this plugin.
https://github.com/DroidKaigi/conference-app-2023/blob/main/build-logic/build.gradle.kts
Or there is a blog reading this. You can refer to it 👍 
https://sys1yagi.hatenablog.com/entry/2023/08/17/151450

Reply:
Thank you for recommendations!
I'm not family with this, so much helpful 🙇

Reply:
I created the Plugin for OSS license plugin.

also It applied for the app module.